.circleci/config.yml

```diff
@@ -260,9 +260,9 @@ jobs:
       - run:
           name: Run visual regression tests
           command: xvfb-run yarn test:regressions
-      - run:
-          name: Upload screenshots to Argos CI
-          command: yarn test:argos
+  #      - run:
```

Comment:
We need to remember to revert it back once we have Argos running again. A `TODO:` comment could have been nice :D

Reply:
We must add the branch protection back then